frontend/assets/koblenz/l10n/override_de.json

```diff
@@ -5,11 +5,14 @@
     "activateDescription": "Sie haben den KoblenzPass bereits beantragt und einen Aktivierungscode erhalten? Scannen Sie den Code hier ein.",
     "activateTitle": "Pass aktivieren",
     "applyDescription": "Sie haben noch keinen KoblenzPass? Hier können Sie Ihren KoblenzPass beantragen.",
-    "cardExpired": "Ihr Pass ist abgelaufen. Unter \"Weitere Aktionen\" können Sie einen Antrag auf Verlängerung stellen.",
+    "cardExpired": "Ihr Pass ist abgelaufen. Unter \"Weitere Aktionen\" können Sie Ihren Pass verlängern oder einen neuen Pass beantragen.",
     "cardInvalid": "Ihr Pass ist ungültig. Er wurde entweder widerrufen oder auf einem anderen Gerät aktiviert.",
     "cardNotYetValid": "Der Gültigkeitszeitraum Ihres Passes hat noch nicht begonnen.",
     "checkFailed": "Ihr Pass konnte nicht auf seine Gültigkeit geprüft werden. Bitte stellen Sie sicher, dass eine Internetverbindung besteht und prüfen Sie erneut.",
     "codeRevoked": "Dieser Pass konnte nicht aktiviert werden, da er widerrufen wurde.",
+    "extendCard": "Pass verlängern",
```

Comment:
It should not be needed to add the translations both to the overrides and the normal translations. As this is quite duplicate, i suggest to remove it.

Reply:
They are different. Normal translation is ”Karte verlängern”, and here is “Pass verlängern”

Reply:
Ah i see, missed that